README.md

```diff
@@ -18,6 +18,24 @@ libraryDependencies ++= Seq(
 )
 ```
 
+## Why log4cats?
+
+Short answer is you don't really need log4cats if you already have logging setup in your app.
```

Comment:
> – Why log4cats?
> – Short answer is you don't really need log4cats...

I'm sorry, but that totally creased me 😆 
Can we change this to something less disappointing for maintainers? 🙇🏻‍♂️

Reply:
Done.  Let me know what you think of the new version :)

Reply:
P.S.  I squashed the commits